src/libraries/sfx-src.proj

```diff
@@ -14,8 +14,7 @@
     <NonNetCoreAppProject Include="@(AnyProject)"
                           Exclude="@(NetCoreAppLibrary->'%(Identity)\src\%(Identity).csproj');
                                    Microsoft.VisualBasic.Core\src\Microsoft.VisualBasic.Core.vbproj" />
-    <ProjectReference Include="@(AnyProject);
-                               $(CoreLibProject)"
+    <ProjectReference Include="@(AnyProject)"
```

Comment:
Why did you change this? CoreLib is part of the sfx-src assemblies build.

Reply:
the sfx trimming breaks in mysterious ways when I don't do this, I'll reach out to you on Teams to debug it :)